packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -212,6 +212,62 @@ export async function fetchCertificate(
   }
 }
 
+/**
+ * Fetch a single destination by name without credentials.
+ * @internal
+ */
+export async function fetchDestinationByNameWithoutTokens(
+  destinationServiceUri: string,
+  serviceToken: string,
+  options: DestinationServiceOptions
+): Promise<Omit<Destination, 'AuthAndExchangeTokens'> | undefined> {
+  const targetUri = `${removeTrailingSlashes(
+    destinationServiceUri
+  )}/destination-configuration/v1/destinations/${
+    options.destinationName
+  }?$skipCredentials=true`;
+
+  if (options?.useCache) {
+    const destinationsFromCache =
+      destinationServiceCache.retrieveDestinationsFromCache(
+        targetUri,
+        decodeJwt(serviceToken)
+      );
+    if (destinationsFromCache && destinationsFromCache.length > 0) {
```

Comment:
[req] You can shorten this + see our [styleguide](https://github.com/SAP/cloud-sdk-js/blob/main/STYLEGUIDE.md#use-truthyfalsy-checks-where-possible).

```suggestion
    if (destinationsFromCache?.length) {
```

Reply:
[q] How can there be more than 1 destination cached for the same key?